cves/kernel/CVE-2021-3178.yml

```diff
@@ -448,7 +478,13 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    This vulnerability existed because of an oversight in the design process.
+    Whichever developer initially designed the code forgot to consider the
+    case where a malicious actor could use '..' to traverse into directories
+    they were not intended to have access to. The fix was just a simple addition
+    of a condition that checks for accessing a parent subdirectory. This simple
+    fix fully addresses the vulnerability. 
```

Comment:
I like this description, it shows how easy it is to make simple mistakes that can lead to a well known vulnerability like path traversal